content/en/blog/2022/k8sattributes.md

```diff
@@ -0,0 +1,255 @@
+---
+title: Improved troubleshooting using k8s attributes
+linkTitle: Kubernetes attributes
+date: 2022-06-23
+author: Ruben Vargas
+spelling: cSpell:ignore k8sattributes K8sattributes K8sprocessor KUBE
+---
+
+Attaching kubernetes resource attributes to traces is something can be very
+useful because it lets you identify which resource (such as a pod) is failing or
+having performance problems. It is also generally useful for correlating across
+other signals.
```

Comment:
```suggestion
Attaching kubernetes resource metadata to OpenTelemetr traces is something can be very
useful because it lets you identify which resource (such as a pod) is failing or
having performance problems. It is also generally useful for correlating across
other signals.
```